lib/reporters/mini.js

```diff
@@ -18,22 +23,45 @@ function MiniReporter() {
 module.exports = MiniReporter;
 
 MiniReporter.prototype.start = function () {
-	return '';
+	var self = this;
+	this.interval = setInterval(function () {
+		self.spinnerFrame = (self.spinnerFrame + 1) % self.spinnerFrames.length;
+		self.write(self.spinnerChar() + self.currentTest);
```

Comment:
@jamestalmage It's not supported on all Terminals, but it dims the spinner to be less intruding. Could maybe also do `chalk.gray.dim()` to support all terminals.